infra/vsts_agent_linux_temp.tf

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+data "template_file" "vsts-agent-linux-startup-temp" {
+  template = "${file("${path.module}/vsts_agent_linux_startup_temp.sh")}"
+
+  vars = {
+    vsts_token   = "${secret_resource.vsts-token.value}"
```

Comment:
I assume this works because it's picked up from the other Terraform file.

Reply:
Yes. Terraform lexical scope is the current folder.